tests/plugins/test_group_manifests.py

```diff
@@ -302,14 +302,14 @@ def mock_registries(registries, config, schema_version='v2', foreign_layers=Fals
 
 
 class Y(object):
-    pass
+    def __init__(self):
+        self.dockerfile_path = None
+        self.path = None
```

Comment:
Alternatively, we could use `stubs.StubSource` instead of `Y`, and `stubs.StubInsideBuilder` instead of `X`.